addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlParser.java

```diff
@@ -50,6 +50,7 @@
     private final ExtensionGraphQl extensionGraphQl =
             Control.getSingleton().getExtensionLoader().getExtension(ExtensionGraphQl.class);
     private final GraphQlParam param = extensionGraphQl.getParam();
+    private boolean originApi;
```

Comment:
I'd suggest `syncGeneration` (or other variation), it's more clear what it is for.